pom.xml

```diff
@@ -23,12 +23,12 @@
     </developers>
 
     <properties>
-        <postmark.version>1.8.0</postmark.version>
+        <postmark.version>1.8.0-jakarta-SNAPSHOT</postmark.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <java.version>1.8</java.version>
         <jackson.minimum.version>2.9.7</jackson.minimum.version>
-        <jackson.version>2.10.0</jackson.version>
```

Comment:
This change may actually be the most important one in this pull request. The current version of jackson being used has two known vulnerabilities, both 7.5/10 severity

[CVE-2020-36518](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-36518)
[CVE-2020-25649](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-25649)

Reply:
this issue is resolved in newer library versions